src/tardev-snapshotter/src/snapshotter.rs

```diff
@@ -914,10 +923,10 @@ impl TarDevSnapshotter {
             let mut store = self.store.write().await;
             extract_dir = store
                 .extract_dir_to_write(&key)
-                .map_err(|e| Status::unknown(format!("failed to create extract dir: {e}")))?;
+                .map_err(|e| Status::unknown(format!("failed to create extract dir: {e:#}")))?;
```

Comment:
What does e:# help do?

Reply:
The anyhow error uses it to concatenate all the contexts.